app/styles.source.css

```
@@ -333,7 +333,7 @@ html.dark {
   }
 
   & pre code {
-    display: inline;
+    display: grid;
```

thanks!

`inline-block` should also suffice

```suggestion
    display: inline-block;
```